src/queries/__tests__/role.test.tsx

```diff
@@ -77,3 +78,61 @@ test('getAllByRole, queryAllByRole, findAllByRole', async () => {
     getNoInstancesFoundMessage(NO_MATCHES_TEXT)
   );
 });
+
+describe('*ByRole with a name', () => {
+  test('Find an element that has the corresponding role and a children with the name', () => {
```

Comment:
nit,style:
Most of the tests for RNTL are named starting with lowercase and using `it()`-like naming, e.g `test('returns ...')`, `test('waits ...')`, etc. Pls adjust the test naming here

Reply:
Woopsy, I'm used to either going `it('should...` or using a fully descriptive name with `test('...`.